pyiron_contrib/atomistics/fitsnap/implementation.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea8942f4-dea7-4265-9ce2-d7d7f7b3a77e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyiron import Project\n",
+    "import numpy as np\n",
+    "import matplotlib.pylab as plt\n",
+    "import json\n",
+    "from pandas import DataFrame\n",
+    "import os\n",
+    "%config InlineBackend.figure_format = 'retina'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b24b04e5-c8e6-4fd1-bdf8-e314ecf3bca6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pr = Project('SNAP')"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "9747bf6b-3087-4703-b602-b7254a0d38fd",
+   "metadata": {},
+   "source": [
+    "plt.xlabel('Steps')\n",
+    "plt.ylabel('Energy eV')\n",
+    "plt.grid()\n",
+    "plt.plot(lmp.output.energy_pot)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc9a6598-9ea2-47a4-8f0d-5611974a4a9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lmp = pr.create.job.Lammps('lmp_Ta')\n",
+    "lmp.structure = pr.create.structure.bulk('Ta', cubic=True).repeat(4)\n",
+    "lmp.calc_md(temperature=300, n_ionic_steps=10000)\n",
+    "lmp.run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e480f58a-12a0-45f0-ab78-44fc93fbeece",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyiron_atomistics.atomistics.structure.atoms import pyiron_to_ase\n",
+    "from ase.calculators.singlepoint import SinglePointCalculator\n",
+    "import ase\n",
+    "def get_ase_structure(job, frame=-1):\n",
+    "    structure = pyiron_to_ase(job.get_structure(frame))\n",
+    "    args = {}\n",
+    "    try:\n",
+    "        args['forces'] = job['output/generic/forces'][frame]\n",
+    "        args['stress'] = -job['output/generic/pressures'][frame].flatten()[[0, 4, 8, 5, 2, 1]] * 1e4\n",
+    "        args['energy'] = job['output/generic/energy_pot'][frame]\n",
+    "    except:\n",
+    "        pass\n",
+    "    calc = SinglePointCalculator(structure, **args)\n",
```

Comment:
Using an ASE calculator should work directly with our `Atoms`.  At least I'm doing it with the `ACE` calculator without problems.  If the conversion is necessary we should look into it.